### PR TITLE
Fix: [IGL-100] Disclosure isExpanded issues

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: packages/Disclosure/src/Disclosure.stories.tsx [hunk 1]
```diff
@@ -88,3 +86,56 @@ export const LowContrast = {
...
+        id: 2,
+        title: "Send Good Vibes 2",
+        description:
+            "Celebrate a team member with a thoughtful card that makes their day",
+    },
+];
+
+export const VerticallyStacked  = {
+    render: () => {
+        const [selected, setSelected] = React.useState<number | null>(null);
```
Comment (alexasselin008) [inline packages/Disclosure/src/Disclosure.stories.tsx:108]: selectedIndex would be a better name here

File: packages/Disclosure/src/Disclosure.tsx [hunk 2]
```diff
@@ -1,129 +1,132 @@
...
-  const classes = cx('ids-disclosure', className, {
-    'ids-disclosure--low-contrast': isLowContrast,
-  });
+    const { buttonProps } = useToggleButton(
+        {},
+        state,
+        btnRef
+    );
+    
+    console.log("Disclosure.tsx: Disclosure: expanded: ", state.isSelected);
```
Comment (alexasselin008) [inline packages/Disclosure/src/Disclosure.tsx:71]: can be removed now

Comment (vicky-comeau) [reply packages/Disclosure/src/Disclosure.tsx:71]: ahh good catch

File: packages/Disclosure/src/Disclosure.stories.tsx [hunk 3]
```diff
@@ -88,3 +86,56 @@ export const LowContrast = {
...
+
+        const handleDisclosureClose = (id: number) => {
+            setSelectedIndex(null);
+        };
+
+
+        return (
+            <div style={{ display: "flex", flexDirection: "column", gap: "1.5rem" }}>
+                {data.map((item, index) => {
+                const isExpanded = selectedIndex === index;
```
Comment (alexasselin008) [inline packages/Disclosure/src/Disclosure.stories.tsx:122]: indentation doesn't seems right

**Review by alexasselin008 — APPROVED** [review]

**Review by franckgaudin — APPROVED** [review]

**Pull request merged**